packages/react-native-session-replay/android/build.gradle

```diff
@@ -167,6 +167,7 @@ dependencies {
     api "com.facebook.react:react-android:$reactNativeVersion"
   }
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+  implementation "com.datadoghq:dd-sdk-android-session-replay:2.0.0"
```

Comment:
the latest version is `2.2.0`

Reply:
I want to keep in sync with the version used in `core`, which is still `2.0.0`
Next week I'm planning to bump to `2.2.0` for the release of RN SDK v2, is it ok for you to keep it at 2.0.0 for now?

Reply:
yes, sure.